.default.env

```diff
@@ -1,5 +0,0 @@
-EXTEND_ESLINT=true
-REACT_APP_ENVIROMENT=stage
-REACT_APP_APPTITLE=ADMIN.EMPLEA.DO
-REACT_APP_PROXY_URL=https://cors-anywhere.herokuapp.com
-REACT_APP_API_URL=https://api-emplea-do.herokuapp.com
```

Comment:
This file served as an example to configure a .env in development environments. Why did you decide to remove it? 😨

Reply:
All the constants were  giving undefined and just renamed the file to `.env` but if that file it wasnt supposed to be deleted I apologize